webpack/assets/javascripts/react_app/components/SearchBar/index.js

```diff
@@ -1,18 +1,27 @@
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
+import uuid from 'uuid';
```

Comment:
Because it is long? though it really allows us to use as many autocomplete reducers as we want.

Reply:
we can use a logic key, for example a string composition with the controller would fit IMH.

Reply:
The controller can change even in the same page though, and many autocompleters can have the same controller,
I passed the uuid into the autocomplete input's `data-autocomplete-id` attribute so it will be easy to find.

another option can be to set a counter of the reducers amount in the autocomplete-reducer-container and call the reducer - `autocomplete-1/2/3....`
but I am also ok with the current situation.

Reply:
ID is required now when mounting the component,
so the consumer can personalize it.